SmartEVSE-3/src/evse.cpp

```diff
@@ -4811,6 +4822,67 @@ void handleWIFImode() {
     }    
 }
 
+void ocppInit() {
```

Comment:
I would love it if you would put your code in #if OCPP  ......  #endif blocks, so we can decide compile time: -DOCPP=1 or -DOCPP=0 . I prefer this over #ifndef....

Reply:
Sure, I can also imagine introducing a new PlatformIO environment for that adding MicroOcpp in the lib_ignore section and not compiling it into the firmware at all.